code/modules/projectiles/guns/projectile/saw.dm

```diff
@@ -55,6 +55,9 @@
 
 
 /obj/item/weapon/gun/projectile/automatic/l6_saw/attackby(obj/item/A, mob/user, params)
+	. = ..()
+	if(.)
```

Comment:
if ..() returns 1 we stop, 1 is returned when an automatic weapon gets a "tactical reload"